MPChartLib/src/com/github/mikephil/charting/charts/Chart.java

```diff
@@ -572,14 +572,12 @@ public void highlightValue(int xIndex, int dataSetIndex) {
 
     /**
      * Highlights the values represented by the provided Highlight object
+     * This DOES NOT generate a callback to the OnChartValueSelectedListener.
      *
      * @param highlight contains information about which entry should be highlighted
      */
     public void highlightValue(Highlight highlight) {
-        if (highlight == null)
-            highlightValues(null);
-        else
-            highlightValues(new Highlight[]{highlight});
+        highlightValue(highlight);
```

Comment:
@PhilJay  Oh man I forgot to add a `, false` in there! Could you do a quick fix?